.circleci/config.yml

```diff
@@ -0,0 +1,47 @@
+version: 2
+jobs:
+  build:
+    resource_class: xlarge
+    docker:
+      - image: kickstarter/android-ci:24ecf62
+        auth:
+          username: ksrci  # can specify string literal values
+          password: $DOCKERHUB_PASSWORD
+    environment:
+      ADB_INSTALL_TIMEOUT: 8
+      GRADLE_OPTS: '-Dorg.gradle.jvmargs="-Xmx512m -XX:+HeapDumpOnOutOfMemoryError"'
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-android-{{ .Branch }}
+            - v1-android-d
+
+      - run: $ANDROID_HOME/tools/android list target
+      - run: make bootstrap-circle
+      - run: echo y | android update sdk --no-ui --all --filter android-25
```

Comment:
> `echo y`

nice

Reply:
yep its a fun trick, there is also `yes | some_command`, yes will repeatedly send `y` so its nice if you need to accept multiple licenses or similar.